Solutions/Marain.Workflow.Api.Services/Marain/Workflows/Api/Services/EngineService.yaml

```diff
@@ -173,7 +175,8 @@ components:
         type: string
     workflowInstanceId:
       name: workflowInstanceId
-      in: "path"
+      in: path
+      x-ms-parameter-location: method
```

Comment:
Curious to know what this is about!

Reply:
None of the other usages of path in the file had quotes...

Reply:
I meant the `x-ms-parameter-location` bit—I don't know what it's for.

Reply:
Oh right! 

It tells AutoRest that the path should be a parameter on the method not a property on the client.

Reply:
Ah—makes sense. Thanks!